content/docs/instrumenting/exporters.md

```diff
@@ -41,52 +41,71 @@ and maintained. We encourage the creation of more exporters but cannot
-   * [BIG-IP exporter](https://github.com/ExpressenAB/bigip_exporter)
-   * [BIND exporter](https://github.com/digitalocean/bind_exporter)
-   * [Ceph exporter](https://github.com/digitalocean/ceph_exporter)
-   * [Cloud Foundry Firehose exporter](https://github.com/cloudfoundry-community/firehose_exporter)
    * [CouchDB exporter](https://github.com/gesellix/couchdb-exporter)
-   * [Django exporter](https://github.com/korfuri/django-prometheus)
+   * [MongoDB exporter](https://github.com/dcu/mongodb_exporter)
+   * [PgBouncer exporter](http://git.cbaines.net/prometheus-pgbouncer-exporter/about)
+   * [PostgreSQL exporter](https://github.com/wrouesnel/postgres_exporter)
+   * [ProxySQL exporter](https://github.com/percona/proxysql_exporter)
+   * [Redis exporter](https://github.com/oliver006/redis_exporter)
+   * [RethinkDB exporter](https://github.com/oliver006/rethinkdb_exporter)
+   * [SQL query result set metrics exporter](https://github.com/chop-dbhi/prometheus-sql)
+
+### Hardware-related
+   * [IoT Edison exporter](https://github.com/roman-vynar/edison_exporter)
+   * [IPMI exporter](https://github.com/lovoo/ipmi_exporter)
+   * [knxd exporter](https://github.com/RichiH/knxd_exporter)
+   * [Ubiquiti UniFi exporter](https://github.com/mdlayher/unifi_exporter)
+
+### Messaging-Systems
+   * [NATS exporter](https://github.com/lovoo/nats_exporter)
+   * [NSQ exporter](https://github.com/lovoo/nsq_exporter)
+   * [RabbitMQ exporter](https://github.com/kbudde/rabbitmq_exporter)
+   * [RabbitMQ Management Plugin exporter](https://github.com/deadtrickster/prometheus_rabbitmq_exporter)
+   * [Mirth Connect exporter](https://github.com/vynca/mirth_exporter)
+
+### Storage
+   * [Ceph exporter](https://github.com/digitalocean/ceph_exporter)
+   * [ScaleIO exporter](https://github.com/syepes/sio2prom)
+
+### Webserver- and proxies
+   * [Apache exporter](https://github.com/neezgee/apache_exporter)
+   * [Nginx metric library](https://github.com/knyar/nginx-lua-prometheus)
+   * [Passenger exporter](https://github.com/stuartnelson3/passenger_exporter)
+   * [Varnish exporter](https://github.com/jonnenauha/prometheus_varnish_exporter)
+
+### APIs
    * [Docker Hub exporter](https://github.com/infinityworksltd/docker-hub-exporter)
    * [GitHub exporter](https://github.com/infinityworksltd/github-exporter)
+   * [OpenWeatherMap exporter](https://github.com/RichiH/openweathermap_exporter)
+   * [Rancher exporter](https://github.com/infinityworksltd/prometheus-rancher-exporter)
+
+### Logging
    * [Google's mtail log data extractor](https://github.com/google/mtail)
    * [Grok exporter](https://github.com/fstab/grok_exporter)
+   
+### Metrics-bridges
+   * [Cloud Foundry Firehose exporter](https://github.com/cloudfoundry-community/firehose_exporter)
+   * [scollector exporter](https://github.com/tgulacsi/prometheus_scollector)
+   * [Munin exporter](https://github.com/pvdh/munin_exporter)
+
+### Services
```

Comment:
If mail is a network service then HTTP is also a network service. Network services is an improvement, but maybe the category itself is not a so good idea.